app/code/Magento/GoogleAnalytics/Block/Ga.php

```diff
@@ -120,6 +120,7 @@ public function getOrdersTrackingCode()
         $result[] = "ga('require', 'ec', 'ec.js');";
 
         foreach ($collection as $order) {
+            $result[] = "ga('set', 'currencyCode', '" . $order->getBaseCurrencyCode() . "');";
```

Comment:
Looks like `BaseCurrencyCode` is not exactly the currency code needed here. With the multi-store setup it will always be the same, regardless of different display currency.